src/client/dfuse/il/int_posix.c

```diff
@@ -291,6 +322,8 @@ ioil_fini(void)
 	DFUSE_TRA_DOWN(&ioil_iog);
 	vector_destroy(&fd_table);
 
+	ioil_show_summary();
```

Comment:
If we have a very long running job, we are going to have to wait until the end to know whether the IL is really working. My original idea was to have a number in this env variable and print to stderr (with a header to differentiate from regular APP output) the first N successful interception and then a summary at the end.